experimental/devices/pca9685/pins.go

```diff
@@ -0,0 +1,115 @@
+package pca9685
```

Comment:
Here too

Reply:
added in https://github.com/google/periph/pull/437/commits/a682bff12501065e21d972c8a9dc029f46777a36